ecosystem/typescript/sdk/src/tests/e2e/client.test.ts

```diff
@@ -1,178 +1,235 @@
 import { AptosApiError, aptosRequest } from "../../client";
 import { VERSION } from "../../version";
-import { NODE_URL } from "../unit/test_helper.test";
+import { getTransaction, longTestTimeout, NODE_URL } from "../unit/test_helper.test";
 
-test("server response should include cookies", async () => {
-  try {
-    const response = await aptosRequest({
-      // use devnet as localnet doesnt set cookies
-      url: "https://fullnode.devnet.aptoslabs.com/v1",
-      method: "GET",
-      originMethod: "test cookies",
-    });
-    expect(response.headers).toHaveProperty("set-cookie");
-  } catch (error: any) {
-    // should not get here
-    expect(true).toBe(false);
-  }
-});
+test(
+  "server response should include cookies",
+  async () => {
+    try {
+      const response = await aptosRequest({
+        // use devnet as localnet doesnt set cookies
+        url: "https://fullnode.devnet.aptoslabs.com/v1",
+        method: "GET",
+        originMethod: "test cookies",
+      });
+      expect(response.headers).toHaveProperty("set-cookie");
+    } catch (error: any) {
+      // should not get here
+      expect(true).toBe(false);
+    }
+  },
+  longTestTimeout,
```

Comment:
Curious, why do we include a longTestTimeout on every test now?

Reply:
it is flaky on how much time it takes for the test to complete, if you notice we use it on most of the tests we just didn't have it on this file